fix(build): force parity of building with SVGR in storybook and gatsby

georgiee opened the pull request:
This will fix the build in the context of SVGR. @feedm3 reported that the Gatsby build can't use the Storybook export as the exports svg files are not matching. Storybook uses the `default` export, while Gatsby creates two exports `default` and `ReactComponent`. Actually the config is virtually matching, this is expected behavior by SVGR when svg files are processed by multiple loaders. See https://react-svgr.com/docs/webpack/#use-with-url-loader-or-file-loader.

We don't have two loaders in storybook, hence the default export.
The fix is forcing SVGR to always export. Together with the fix I can use `<Illustration/>` in both worlds as expected ✅

Remarks: This requires SVGR in version 6 which was released just 12 hours ago. Only there you have the feature to force the named export. See the [changelog](https://github.com/gregberge/svgr/releases/tag/v6.0.0). An alternative workaround would have been to add a second loader to match the Gatsby environment, which we might want to do in the future to have a full parity of build systems anyway? Good for now.

Thanks for reporting @feedm3 🙌

georgiee committed:
fix(build): force parity of building with SVGR in storybook and gatsby

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**satellytes.com-main**

:tada: Your build was successful! [See the Deploy preview here.](https://satellytescommain21751-gkfixillustration.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/674be368-adf0-4ac5-8fae-f9a4493adc51/builds/ac39b6fd-f62e-4ce8-a86c-f34c7f4e5464/details)

:clock1: Build time: **6m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 72   |
| Accessibility |   :green_heart: 97   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 85   |

:link: [View full report](https://build-ac39b6fd-f62e-4ce8-a86c-f34c7f4e5464.gtsb.io/reports/lighthouse/index.html)

georgiee committed:
chore: fix accordion direct usage of svg

georgiee commented:
There was a build error:
> Error: src/components/accordion/accordion-header.tsx(55,8): error TS2604: JSX element type 'ChevronSVG' does not have any construct or call signatures.

The chevron is a direct import in the accordion header and relied on the default export. I fixed it and updated the PR.

georgiee committed:
Merge branch 'main' into gk-fix-illustration
